platform/ios/main.m

```diff
@@ -46,7 +46,7 @@ int main(int argc, char *argv[]) {
 	gargv = argv;
 
 	@autoreleasepool {
-		NSString *className = NSStringFromClass([GodotApplicalitionDelegate class]);
+		NSString *className = NSStringFromClass([GodotApplicationDelegate class]);
```

Comment:
The same should be changed in the [platform/ios/godot_app_delegate.h](https://github.com/godotengine/godot/blob/master/platform/ios/godot_app_delegate.h) and [platform/ios/godot_app_delegate.m](https://github.com/godotengine/godot/blob/master/platform/ios/godot_app_delegate.m).